frame/staking/src/pallet/impls.rs

```diff
@@ -540,14 +556,14 @@ impl<T: Config> Pallet<T> {
 			);
 		}
 
-		elected_stashes
+		elected_stashes.try_into().expect("since we only map through exposures, size of elected_stashes is always same as exposures; qed")
```

Comment:
better make this conversion at the top of the function next to the `map`, so the proof is clearer.